docs/Report.md

```diff
@@ -1408,15 +1437,17 @@ We can also see if we query in pgAdmin that ouur new user has been generated, an
 
 * We show similar books when searching by any given parameter. You will always return the top 5 matches, assuming there are 5, for any given search.
 
-* We implemented a sense of "state" throughout the client side of the aplication. Once you go to checkout ffor the first time, you will be promted to either sign in or create a new account. After doing so, your credentials will be stored such that you can navigate around the store in any direction you want, and when it comes time to checkout again, you will not need to provide your information for  a second time. We keep track of the logged in users credentials, the connection point to the database stored on the hostss computer, and the cart currently in action through out the alication. 
+* We implemented a sense of "state" throughout the client side of the aplication. Once you go to checkout for the first time, you will be promted to either sign in or create a new account. After doing so, your credentials will be stored such that you can navigate around the store in any direction you want, and when it comes time to checkout again, you will not need to provide your information for  a second time. We keep track of the logged in users credentials, the connection point to the database stored on the hostss computer, and the cart currently in action through out the alication. 
 
 * Rather then displaying a single Order at a time given an input order number, we designed the application to all of the previous reports, including their dates, total prices, current location, and fulfilment status. This is more akin to a real e-commerce store where you can view and scroll through all of your past purchasses. 
 
+* In order to expand on the Owner side, we added the concept of order 'state' and fulfillment. So, when an order gets placed, it is not instantly sent out. The owner will actually have to go through the menu, and manually 'fulfill' an order. Once an order is 'fulfilled', it will get a new status, it will store which owner shipped it, and the location will be updated (through the Faker gem giving it a random location). This adds more realism to the project.
 
+* To generate the values for the Database, we do not hardcode the values, making it easier to see a wide array of functionality using different values, and with more values. Using random generation also allows the user to super easily change how many values are generated for a given table, allowing a user much more control on the data-set they are working with. It was a challenge to tame randomness, as, well, it is random. Randomness introduced some edge cases where we potentially could have duplicate unique values, but we have set up edge case detection to ensure that this does not happen.
```

Comment:
This whole paragraph is not needed. You have explained the gem and the randomness and the error detection already. This is repetitive.